pdata/testdata/profile.go

```diff
@@ -38,6 +38,8 @@ func fillProfileOne(profile pprofile.ProfileContainer) {
 	profile.SetEndTime(profileEndTimestamp)
 	profile.SetDroppedAttributesCount(1)
 
+	profile.Profile().LocationIndices().FromRaw([]int64{1})
```

Comment:
Changing the profiles testdata is probably what is making all the other tests fail.

Reply:
For other profiles, the testdata remains fairly simple. And tests that need more create their own data (possibly with a testdata start).
Doing that would avoid having to make changes everywhere to let the tests pass.

Reply:
Thank you! Moved it to its own method.

Reply:
I'll let @mx-psi chime in, but I think it'd be better if this method were in the otlptext package, as pdata is public and therefore has much stricter stability requirements.

Reply:
> I think it'd be better if this method were in the otlptext package

Agreed, better to make it internal 

>  pdata is public and therefore has much stricter stability requirements

In this case the `pdata/testdata` folder is its own module so it wouldn't be as bad, but still if the only user is `otlptext` then it should be in `otlptext`

Reply:
moved to `otlptext`, thank you @dmathieu and @mx-psi ⭐